docker-init.sh

```diff
@@ -1,7 +1,12 @@
 #!/bin/sh
 echo "prepare environment"
 # replace BASE_URL constant by REMARK_URL
-sed -i "s|https://demo.remark42.com|${REMARK_URL}|g" /srv/web/*.js
+sed -i "s|https://demo.remark42.com|${REMARK_URL}|g" /srv/web/*.{js,html}
```

Comment:
We don't need to `sed` in HTML files because we already have the right URL throw env variable.
Correct me if I'm wrong.

Reply:
After my changes final static html files in docker image contain constant value for `host` in `<script>` tag. And constant value `remarkUrl` in `webpack.config.js` is `https://demo.remark42.com`.

Reply:
I changed templates and replaced `host: window.location.origin` to `host: <%= REMARK_URL %> || window.location.origin`.
`REMARK_URL` in templates wll be replaced by webpack. In webpack config this variable filled by `remarkUrl`.

Reply:
yep, you can change this behavior and replace `remarkUrl` value to process.env.REMARK_URL